iphone/Classes/TiBluetoothCentralManagerProxy.m

```diff
@@ -211,6 +227,22 @@ - (TiBluetoothPeripheralProxy *)peripheralProxyFromPeripheral:(CBPeripheral *)pe
     return result;
 }
 
+- (NSArray<TiBluetoothPeripheralProxy *> *)peripheralProxyArrayFromPeripheralArray:(NSArray<CBPeripheral *> *)peripherals
+{
+    if (peripherals == nil) {
+        return nil;
+    }
+    
+    NSMutableArray<TiBluetoothPeripheralProxy*> *result = [NSMutableArray array];
+    
+    for (id peripheral in peripherals) {
```

Comment:
Change to `CBPeripheral  *peripheral`, since the type is determined by the array template-type